core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha.bp

```diff
@@ -8,64 +8,45 @@
     "displayName": "ha-datalake",
     "hostTemplates": [
       {
+        "cardinality": 2,
         "refName": "master",
         "roleConfigGroupsRefNames": [
-          "hbase-REGIONSERVER-BASE",
-          "hbase-MASTER-BASE",
           "hdfs-DATANODE-BASE",
           "hdfs-FAILOVERCONTROLLER-BASE",
           "hdfs-JOURNALNODE-BASE",
-          "hdfs-NAMENODE-1",
+          "hdfs-NAMENODE-BASE",
           "hive-HIVEMETASTORE-BASE",
-          "solr-SOLR_SERVER-BASE",
-          "zookeeper-SERVER-BASE"
-        ]
-      },
-      {
-        "refName": "master2",
-        "roleConfigGroupsRefNames": [
-          "hdfs-BALANCER-BASE",
-          "hbase-REGIONSERVER-BASE",
-          "hbase-MASTER-BASE",
-          "hdfs-DATANODE-BASE",
-          "hdfs-FAILOVERCONTROLLER-BASE",
-          "hdfs-JOURNALNODE-BASE",
-          "hdfs-NAMENODE-2",
-          "zookeeper-SERVER-BASE"
+          "zookeeper-SERVER-BASE",
```

Comment:
In new setup there are 4 ZooKeeper servers, but their number should be odd.  Can you also add it to `gateway` group?  Depending on whether you want 3 or 5 ZK servers, optionally remove it from either `master` or `alpha`, or keep in both.

Reply:
ok

Reply:
updated the pull request.